ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -88,13 +89,18 @@ Set Server Size
 
 Verify GPU Selector Is Usable
     [Documentation]    Verifies that the GPU selector is present and enabled
-    Page Should Contain Element    ${GPU_SECTION_TITLE}
-    Element Should Be Enabled    ${GPU_SECTION_INPUT}
+    Page Should Contain Element    ${SERVING_ACCELERATOR_DROPDOWN_XPATH}
+
+Set Accelerator
+    [Documentation]  Set NVidia GPU Accelerator
+    [Arguments]    ${accelerator}='Nvidia GPU'
+    Click Element  ${SERVING_ACCELERATOR_DROPDOWN_XPATH}
+    Click Element  xpath=//div[@class and text()=${accelerator}]
 
 Set Number of GPU With Buttons
```

Comment:
should this kw be valid for any accelerators?

Reply:
Yes, as long as you have the displayed name of the Accelerator

Reply:
okay then could you rename it and change documentation in order to make it generic?

Reply:
Fixed in [7e13d55](https://github.com/red-hat-data-services/ods-ci/pull/971/commits/7e13d55eb6a7ea956d90c1cd44cb20b1cf47fbc1)

Reply:
i thought accelerator could be also different from GPUs